Apollo.Dockerfile

```
@@ -0,0 +1,30 @@
+FROM node:22-alpine3.20
+
+RUN apk add --no-cache curl
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)